mlrun/runtimes/databricks_runtime_wrapper.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2023 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+def run_mlrun_databricks_job(
+    context,
+    internal_code,
+    token_key="DATABRICKS_TOKEN",
+    timeout=20,
```

Comment:
I think this needs to be called `timeout_minutes`, or else be changed to seconds (standard unit of duration in python). Probably changing this to seconds is best for more granular control.